scripts/calendar/mongolian_generator.py

```
@@ -0,0 +1,204 @@
+    d = int_div(h % 153, 5) + 1
+    m = (int_div(h, 153) + 2) % 12 + 1
+    y = int_div(e, 1461) - 4716 + int_div(14 - m, 12)
+    return {"year": y, "month": m, "day": d}
+
+
+def mongolian_to_gregorian(m_year, m_month, is_leap, m_day):
+    jd = julian_day(m_year, m_month, is_leap, m_day)
+    return gregorian_date(jd)
+
+
+def get_tsagaan_sar(y):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)